lib/WebpackConfig.js

```diff
@@ -750,6 +752,25 @@ class WebpackConfig {
         this.loaderConfigurationCallbacks[name] = callback;
     }
 
+    enableIntegrityHashes(enabled = true, algorithms = ['sha384']) {
```

Comment:
should we also forbid passing `enabled = true` with an empty array as second argument ? That would actually disable it.